src/libraries/Common/tests/System/Security/Cryptography/PlatformSupport.cs

```diff
@@ -129,5 +159,6 @@ private static bool CheckIfVbsAvailable()
 
         private static bool? s_isVbsAvailable;
         internal static bool IsVbsAvailable => s_isVbsAvailable ??= CheckIfVbsAvailable();
+        internal static bool IsRsaPssSupported { get; } = CheckIfRsaPssSupported();
```

Comment:
This is going to eagerly check if RSA-PSS is supported in the static constructor which affects test discovery times. We should do this lazily like we do for `IsVbsAvailable`.